### PR TITLE
NFC: Small corrections in move-mv-llvm-compiler::cli

### PR DESCRIPTION
**Pull request by jcivlin**
Small corrections, in particular, a passed function parameter should be an Option rather than Result.

Also VS Code config is removed.

### CODE REVIEW

**Comment by ksolana** [comment]
> Also VS Code config is removed.

any reason we want to remove this?

**Comment by dmakarov** [comment]
https://github.com/solana-labs/move/actions/runs/6056987034/job/16437652599

**Comment by jcivlin** [comment]
> > Also VS Code config is removed.
> 
> any reason we want to remove this?

It should be local only. One that I use is very different and I need to maintain it manually because there is one in the repo.

File: language/tools/move-mv-llvm-compiler/src/main.rs [hunk 1]
```diff
@@ -54,30 +54,26 @@ fn main() -> anyhow::Result<()> {
     // and cli may move cwd while resolving them.
     let mut output_file_path = args.output_file_path.clone();
     if output_file_path != "-" {
-        let out_absolute = absolute_path(
-            &Some(output_file_path.to_string()),
+        let out_absolute = absolute_new_file(
+            Some(output_file_path.to_string()),
             &"output_file_path".to_string(),
```
Comment (dmakarov) [inline language/tools/move-mv-llvm-compiler/src/main.rs]: Fix clippy errors.

https://github.com/solana-labs/move/actions/runs/6056987034/job/16437652599

**Review by dmakarov — CHANGES_REQUESTED** [review]

File: language/tools/move-mv-llvm-compiler/src/main.rs (hunk 1 above)
Comment (jcivlin) [reply language/tools/move-mv-llvm-compiler/src/main.rs]: Fixed. Thx!

**Review by dmakarov — APPROVED** [review]

**Pull request merged**